tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java

```diff
@@ -4593,12 +4593,47 @@ public void testCoverage() {
                 "en Modern",
                 validShortNoAliases,
                 Sets.difference(enModern, aliasedShortUnits.keySet()));
+        final Set<String> enOrJaOnly =
+                Set.of(
+                        "bu-jp",
+                        "cho",
+                        "se-jp",
+                        "fortnight",
+                        "british-thermal-unit",
+                        "british-thermal-unit-it",
+                        "chain",
+                        "fathom",
+                        "furlong",
+                        "jo-jp",
+                        "ken",
+                        "ri-jp",
+                        "rin",
+                        "rod",
+                        "shaku-cloth",
+                        "shaku-length",
+                        "sun",
+                        "fun",
+                        "slug",
+                        "stone",
+                        "gasoline-energy-density",
+                        "rankine",
+                        "bushel",
+                        "cup-imperial",
+                        "cup-jp",
+                        // "fluid-ounce-metric",
```

Comment:
"fluid-ounce-metric" as a comment?

Reply:
It was dropped from that list. Again, left in case we have to restore.